tests/test_linearize.py

```diff
@@ -65,10 +92,10 @@ def reports(
     for uuid, (start_time, end_time) in zip(uuids, intervals):
         # reports from the same worker always have monotonically increasing
         # coverage, timestamps, and elapsed_time
-        # (TODO: we don't actually rely on timestamps being ordered. But our logic
-        # for unordered reports isn't correct yet either (we drop them, when we
-        # should correctly recompute/reinsert). Drop the .map(sorted) for timestamps
-        # when we implement this correctly.
+        # (TODO: we shouldn't actually rely on timestamps being ordered. But
+        # test_single_worker fails without it. This might be a real failure
+        # or it might be an acceptable edge case. Drop the .map(sorted) for timestamps
+        # when we investigte and fix it / figure out which it is.
```

Comment:
I'd prefer to at least understand what's going on before we merge; OK to defer if it's a rare-seeming edge case but shipping a bug would be sad.

Reply:
whew, was a problem with the test itself (sorting by .timestamp instead of .elapsed_time).